python/docs/source/index.md

```diff
@@ -3,94 +3,116 @@ hide-toc: true
 html_theme.sidebar_secondary.remove: true
 title: Kaskada Timestreams
 ---
+# Introduction
 
 <div class="px-4 py-5 my-5 text-center">
     <img class="d-block mx-auto mb-4" src="_static/kaskada.svg" alt="" width="auto">
-    <h1 class="display-5 fw-bold">Event-processing for AI applications.</h1>
+    <h1 class="display-5 fw-bold">Real-Time AI without the fuss.</h1>
     <div class="col-lg-7 mx-auto">
-      <p class="lead mb-4">Next-generation, real-time and historic event processing.
+      <p class="lead mb-4">Kaskada is a next-generation streaming engine that connects AI models to real-time & historical data.
       </p>
     </div>
 </div>
 
+## Kaskada completes the Real-Time AI stack, providing...
+
 ```{gallery-grid}
 :grid-columns: 1 2 2 3
 
-- header: "{fas}`timeline;pst-color-primary` Real-time processing for all"
-  content: "Quickly process events so you can respond in real-time."
-  link: ".#stream"
-- header: "{fab}`python;pst-color-primary` Python-native"
-  content: "Use Python so you can load data, process it, and train and serve models from one place."
-  link: ".#python"
-- header: "{fas}`gauge-high;pst-color-primary` Get started immediately"
-  content: "No infrastructure to provision let's you jump right in."
-  link: ".#get-started"
-
-- header: "{fas}`fast-forward;pst-color-primary` Real-time, Batch and Streaming"
-  content: "Execute large-historic queries or materialize in real-time. Or both."
-  link: ".#real-time-and-historic"
-- header: "{fas}`rocket;pst-color-primary` Local, Remote and Distributed"
-  content: "Develop and test locally. Deploy to Docker, K8s or a service for production."
-  link: ".#local-and-distributed"
-- header: "{fas}`backward;pst-color-primary` Time-travel"
-  content: "Generate training examples from the past to predict the future."
-  link: ".#time-travel"
+- header: "{fas}`timeline;pst-color-primary` Real-time Aggregation"
+  content: "Precompute model inputs from streaming data with robust data connectors, transformations & aggregations."
+- header: "{fas}`binoculars;pst-color-primary` Event Detection"
+  content: "Trigger pro-active AI behaviors by identifying significant changes in your data, when they happen."
```

Comment:
Significant events may not be changes? Not sure what would be better though ("occurrences"?)

Reply:
"Occurrences" feels wordy to me. I was going to say that all events are changes in your data, but I suppose maybe timers aren't data per se? We could go with "identifying significant events" - I didn't use that originally because it suggests that we can only react to input events.

Reply:
Going to try "Trigger pro-active AI behaviors by identifying important activies, as they happen". Starting to feel like word salad - think I've been looking at it too much.